### PR TITLE
Outfit and Access Spring Cleaning

### LINKED ISSUE #11788: xenobotany doesn't have access to strange seeds from cargo
The crate is access locked and xenobotany can't open it. That seems kind of silly because the strange seeds are typically for xenobotany.

### LINKED ISSUE #13869: Xenobotanist Zeng-Hu card does not work correctly
The Zeng-Hu Xenobotanist card is coded incorrectly. It gives access to the RnD Lab, but not to the Xenobotany one.
The NT Xenobotanist card works, however.

### PR DESCRIPTION
**Pull request by JohnWildkins**
changes:
  - bugfix: "Fixed secure crates starting unlocked."
  - bugfix: "Secure hydroponics crates now require either Hydroponics or Xenobiology access."
  - bugfix: "Xenobotanists now spawn with the correct access and job outfit."
  - bugfix: "Machinists, PMCG medical staff, and Zavod/Zeng-Hu scientists now start with their proper labcoats."
  - bugfix: "Idris gardeners and janitors start with aprons once more."
  - bugfix: "Idris gardener uniform now has proper directional sprites."
  - bugfix: "Security cadets now start with their appropriate hazard vest again."

Fixes #13869
Fixes #11788

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

File: code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -88,17 +88,17 @@
 	icon_state = "labcoat_heph"
 
 /obj/item/clothing/suit/storage/toggle/labcoat/epmc
-	name = "epmc security labcoat"
-	desc = "A suit that protects against minor chemical spills. Comes in EPMC colours."
+	name = "PMCG security labcoat"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/labcoat.dm:91]:
```suggestion
	name = "\improper PMCG security labcoat"
```

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -88,17 +88,17 @@
...
 	icon_state = "labcoat_erisec"
 
 /obj/item/clothing/suit/storage/toggle/labcoat/epmc/alt
 	desc = "A suit that protects against minor chemical spills. Darker than the standard issue."
 	icon_state = "labcoat_erisec_alt"
 
 /obj/item/clothing/suit/storage/toggle/labcoat/epmc/med
-	name = "epmc medical labcoat"
-	desc = "A suit that protects against minor chemical spills. Comes in EPMC colours."
+	name = "PMCG medical labcoat"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/labcoat.dm:100]:
```suggestion
	name = "\improper PMCG medical labcoat"
```

File: html/changelogs/johnwildkins-zengfix.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+author: JohnWildkins
+
+delete-after: True
+
+changes:
+  - bugfix: "Fixed secure crates starting unlocked."
+  - bugfix: "Secure hydroponics crates now require either Hydroponics or Xenobiology access."
+  - bugfix: "Xenobotanists now spawn with the correct access and job outfit."
+  - bugfix: "Machinists, PMCG medical staff, and Zavod/Zeng-Hu scientists now start with their proper labcoats."
+  - bugfix: "Idris gardeners and janitors start with aprons once more."
```
Comment (SleepyGemmy) [inline html/changelogs/johnwildkins-zengfix.yml:10]:
```suggestion
  - bugfix: "Idris gardeners start with aprons once more."
```
janitors never had aprons.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**